### DISCUSSION TITLE
Add `hazard` and `toxicityLevel` fields

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5983] [Feature Requests]
New fields in `ConservDescription`:
* `toxicityLevel` (picklist: low, medium, high || quantitative values) – string
* `hazard` (picklist: toxic, poison, arsenic, explosive, etc.) – string

The hazard should be resolved to one of the [hazard pictograms](https://www.hse.gov.uk/chemical-classification/labelling-packaging/hazard-symbols-hazard-pictograms.htm) when applicable. These should be displayed on the form next to the value selected.

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8178
